tony-history-server/build.gradle

```diff
@@ -0,0 +1,47 @@
+plugins {
+    id 'play'
+    id 'idea'
+}
+
+def playVersion = "2.6.20"
+def hadoopVersion = "3.1.1"
+def scalaVersion = System.getProperty("scala.binary.version", "2.12")
+
+model {
+    components {
+        play {
+            platform play: playVersion, scala: scalaVersion, java: '1.8'
+            injectedRoutesGenerator = true
+
+            sources {
+                twirlTemplates {
+                    defaultImports = TwirlImports.JAVA
+                }
+            }
+        }
+    }
+}
+
+dependencies {
+    play "com.typesafe.play:play-guice_$scalaVersion:$playVersion"
+    play "com.typesafe.play:play-logback_$scalaVersion:$playVersion"
+    play "org.apache.hadoop:hadoop-hdfs-client:$hadoopVersion"
+    play "org.apache.hadoop:hadoop-common:$hadoopVersion"
```

Comment:
Please keep alphabetized.

Also, instead of hardcoding the dependencies here, please define them in the root build.gradle file and reference that, e.g.: `deps.hadoop.common`, `deps.hadoop.hdfs_client`, etc.